### PR TITLE
Fix delete magic

### PR DESCRIPTION
**Pull request by aggFTW**

### CODE REVIEW

**Comment by aggFTW** [comment]
Sorry for crazy commits. Can you squash merge when this passes? Will continue the bug bash later. Need to run out!

File: remotespark/kernels/kernelmagics.py [hunk 1]
```diff
@@ -234,7 +234,7 @@ def cleanup(self, line, cell="", local_ns=None):
     @magic_arguments()
     @cell_magic
     @argument("-f", "--force", type=bool, default=False, nargs="?", const=True, help="If present, user understands.")
-    @argument("-s", "--session", type=str, help="Session id number to delete.")
+    @argument("-s", "--session", type=int, help="Session id number to delete.")
```
Comment (msftristew) [inline remotespark/kernels/kernelmagics.py]: Good catch -- this was a silly mistake.

File: tests/test_kernel_magics.py [hunk 2]
```diff
@@ -531,7 +531,7 @@ def test_cleanup_exception():
 
 @with_setup(_setup, _teardown)
 def test_delete_without_force():
-    session_id = "0"
+    session_id = 0
```
Comment (msftristew) [inline tests/test_kernel_magics.py]: I hate dynamically typed languages

**Comment by msftristew** [comment]
Will do

**Pull request merged**